web/src/main/java/org/mskcc/smile/web/CrosswalkController.java

```diff
@@ -0,0 +1,76 @@
+package org.mskcc.smile.web;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.mskcc.smile.model.web.CrdbCrosswalkTriplet;
+import org.mskcc.smile.service.CrdbMappingService;
+import org.mskcc.smile.service.exception.SmileWebServiceException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@CrossOrigin(origins = "*")
+@RequestMapping(value = "/")
+@Api(tags = "crosswalk-controller", description = "Crosswalk Controller")
+@PropertySource("classpath:/maven.properties")
+public class CrosswalkController {
+
+    @Value("${smile.schema_version}")
+    private String smileSchemaVersion;
+
+    @Autowired
+    private CrdbMappingService crdbMappingService;
+
+    @Autowired
+    public CrosswalkController(CrdbMappingService crdbMappingService) {
+        this.crdbMappingService = crdbMappingService;
+    }
+
+    /**
+     * Returns a list of mrn-dmp-cmo patient id triplets given a list of MRNs.
```

Comment:
To confirm my understanding, technically this REST endpoint will return the triplets given not just MRNs but also DMP IDs and/or CMO Patient IDs, correct? If so, should we update the description to reflect that?

Reply:
Yeah, thats what I mean my mrn-dmp-cmo - thats the triplet.   The class is called CrdbCrosswalkTriplet too.